.circleci/config.yml

```diff
@@ -0,0 +1,51 @@
+# Ruby CircleCI 2.0 configuration file
+#
+# Check https://circleci.com/docs/2.0/language-ruby/ for more details
+#
+version: 2
+jobs:
+  build:
+    docker:
+       - image: circleci/ruby:2.4.1-node-browsers
+       - image: luckyj5/splunk-circleci-hec-720:latest
```

Comment:
remove local repo reference

Reply:
which one? 
Splunk repo is customized for this setup and needed to push it to be able to pull it from CI.

Reply:
luckyj5/splunk-circleci-hec-720:latest - this should be hosted at splunk

Reply:
Thanks @chaitanyaphalak. We already discussed this and it's ok this way for now.